documentation/installation.md

```diff
@@ -40,6 +40,11 @@ pointing to it instead:
 
     ln -s <cloned_repo_path> deploy
 
+
+**Note:** Make sure to use Python 3.5 or 3.6 as the ansible-playbooks are not yet compatible 
+with the newer Python versions. You can use pyvenv to setup Python 3.5 or 3.6. For more information
+see https://hackernoon.com/reaching-python-development-nirvana-bb5692adf30c
```

Comment:
@ahmedbilal, nit: wouldn't it be better to link to the [official README](https://github.com/pyenv/pyenv) instead of some random link on the internet? :) It's okay if you want to retain this and also add the link to the official README file :)

Reply:
@lgp171188 Yep, that logical. Actually, I substituted `virtualenv` in my mind and forgot to mention it.